src/FormsyCheckbox.js

```diff
@@ -63,7 +66,7 @@ class FormsyCheckbox extends Component {
 
     const checkboxProps = {
       ...filterSuirElementProps(this.props),
-      checked: getValue(),
+      checked: checked || defaultChecked,
```

Comment:
We can't react to changes in Formsy's model if we do this. Example: ```this.form.reset()``` will have no effect.